gemini/evaluation/evaluating_prompts_at_scale_with_gemini_batch_prediction_api.ipynb

```diff
@@ -0,0 +1,1011 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "zOKg8sxIZnzm"
+   },
+   "source": [
+    "### Create BigQuery dataset and load table with ground truth"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "ZjWGl6V9ZvhV",
+    "outputId": "fd61b025-da15-448f-a16b-475a6c273638"
+   },
+   "outputs": [],
+   "source": [
+    "def create_dataset(dataset_id, project, location):\n",
+    "    \"\"\"Creates a BigQuery dataset.\"\"\"\n",
+    "    dataset = bigquery.Dataset(dataset_id)\n",
+    "    dataset.location = location\n",
+    "\n",
+    "    dataset = bq_client.create_dataset(dataset, timeout=30)\n",
+    "    print(\"Created dataset {}.{}\".format(bq_client.project, dataset.dataset_id))\n",
+    "\n",
+    "\n",
+    "def load_files_table_from_uri(files_table, uri):\n",
+    "    \"\"\"Load ground truth table from a URI.\"\"\"\n",
+    "    job_config = bigquery.LoadJobConfig(\n",
+    "        schema=[\n",
+    "            bigquery.SchemaField(\"path\", \"STRING\"),\n",
+    "            bigquery.SchemaField(\"label\", \"STRING\"),\n",
+    "        ],\n",
+    "        skip_leading_rows=1,\n",
+    "        source_format=bigquery.SourceFormat.CSV,\n",
+    "        write_disposition=bigquery.WriteDisposition.WRITE_TRUNCATE,\n",
+    "    )\n",
+    "    load_job = bq_client.load_table_from_uri(uri, files_table, job_config=job_config)\n",
+    "    load_job.result()\n",
+    "\n",
+    "    destination_table = bq_client.get_table(files_table)\n",
+    "    print(\"Loaded {} rows.\".format(destination_table.num_rows))\n",
+    "\n",
+    "\n",
+    "create_dataset(dataset_id=BQ_DATASET, project=PROJECT_ID, location=LOCATION)\n",
+    "load_files_table_from_uri(files_table=FILES_TABLE, uri=SPORTS_FILE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "OW4EngZRLlts"
+   },
+   "source": [
+    "### Test image URIs from BigQuery"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 82,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "tRgVfFZIN7ae",
+    "outputId": "f53c1f11-f619-44a4-f9fc-583ebeba2586"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['train/tennis/064.jpg', 'train/tennis/045.jpg']\n"
+     ]
+    }
+   ],
+   "source": [
+    "def get_filepaths(files_table):\n",
+    "    \"\"\"Get filepaths from the ground truth table in BigQuery.\"\"\"\n",
+    "    job = bq_client.query(\n",
+    "        f\"\"\"\n",
+    "      SELECT path\n",
+    "      FROM {files_table}\"\"\"\n",
+    "    )\n",
+    "    return [row[0] for row in job.result()]\n",
+    "\n",
+    "\n",
+    "images_uri = get_filepaths(files_table=FILES_TABLE)\n",
+    "print(images_uri[:2])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
Is this really needed?

Reply:
Updated it to use BigQuery Pandas instead